src/bin/tf.rs

```diff
@@ -1,21 +1,16 @@
-use std::fs::File;
-use std::io::Cursor;
-use std::str::FromStr;
-
 use anyhow::Error;
-use memmap::MmapOptions;
 use regex::Regex;
+use std::path::Path;
```

Comment:
I prefer to keep the `std` imports separate (and these changes are not related to the topic of this commit/PR anyway.

Reply:
Done.